cmd/av/pr_status.go

```diff
@@ -150,6 +159,28 @@ var prStatusCmd = &cobra.Command{
 			)
 		}
 
+		// Get Associated BotPR info
+		botPullRequest := query.GithubRepository.PullRequest.BotPullRequest
+		if botPullRequest.Number == 0 {
+			// no associated botPR
+			return nil
+		}
+
+		_, _ = fmt.Fprint(os.Stderr, "Associated PR #", botPullRequest.Number, " Required Checks\n")
+
+		botRequiredCheckStatuses := query.GithubRepository.PullRequest.BotPullRequest.RequiredCheckStatuses
+		for index := range botRequiredCheckStatuses {
```

Comment:
```suggestion
		for _, status := range botRequiredCheckStatuses {
```

Doing `for i := range slice` yields the indices, but you can also do `for i, elt := range slice` to get the indices and their values (and if you don't need the value, you can replace the `i` with a `_` to discard it). Same works for iterating over maps (first form iterates over keys, second form iterates over keys and values).